types/pdf/index.d.ts

```diff
@@ -474,3 +474,7 @@ interface PDFJSStatic {
 }
 
 declare var PDFJS: PDFJSStatic;
+
+declare module "pdf" {
+	export = { PDFJS };
```

Comment:
instead of an ambient module declaration (i.e. the following)

```ts
declare module "pdf"
```

Replace it with

```ts
export = PDFJS;
export as namespace PDFJS;
```